app/scripts/templates/settings/clients.mustache

```diff
@@ -52,7 +52,7 @@
 
       <div class="button-row">
         <button type="submit" class="settings-button primary clients-refresh">{{#t}}Refresh{{/t}}</button>
-        <button class="settings-button secondary cancel">{{#t}}Done{{/t}}</button>
+        <button class="settings-button secondary cancel" autofocus>{{#t}}Done{{/t}}</button>
```

Comment:
![aaa](https://cloud.githubusercontent.com/assets/128755/22900042/f2da6502-f1fa-11e6-84db-0f32a2f77d2e.gif)

I don't think we can ship this in its current form, we need to use a different event for this panel and not the autofocus approach.